files/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -15,200 +15,46 @@ function Join-Paths($path, $childPaths) {
 Set-Variable 'IS_WINDOWS' -Scope Script -Option Constant -Value $((Get-Variable -Name IsWindows -ValueOnly -ErrorAction Ignore) -or !(Get-Variable -Name IsCoreClr -ValueOnly -ErrorAction Ignore))
 Set-Variable 'EXE_EXT' -Scope Script -Option Constant -Value $(if ($IS_WINDOWS) { '.exe' } else { '' })
 
-<#
-.SYNOPSIS
-Builds a repository
-
-.DESCRIPTION
-Invokes the default MSBuild lifecycle on a repostory. This will download any required tools.
-
-.PARAMETER Path
-The path to the repository to be compiled
-
-.PARAMETER MSBuildArgs
-Arguments to be passed to the main MSBuild invocation
-
-.EXAMPLE
-Invoke-RepositoryBuild $PSScriptRoot /p:Configuration=Release /t:Verify
-
-.NOTES
-This is the main function used by most repos.
-#>
-function Invoke-RepositoryBuild(
-    [Parameter(Mandatory = $true)]
-    [string] $Path,
-    [Parameter(ValueFromRemainingArguments = $true)]
-    [string[]] $MSBuildArgs) {
-
-    $ErrorActionPreference = 'Stop'
-
-    if (-not $PSBoundParameters.ContainsKey('Verbose')) {
-        $VerbosePreference = $PSCmdlet.GetVariableValue('VerbosePreference')
-    }
+function Set-Settings
```

Comment:
`Set-KoreBuildSettings` would be better to avoid possible conflicts with other modules that may use this name.